source/extensions/filters/http/ext_proc/ext_proc.h

```diff
@@ -431,7 +431,7 @@ class Filter : public Logger::Loggable<Logger::Id::ext_proc>,
     return config_with_hash_key_.config();
   }
 
-  ExtProcFilterStats& stats() { return stats_; }
+  const ExtProcFilterStats& stats() { return stats_; }
```

Comment:
can this be:

const ExtProcFilterStats& stats() const ?

Reply:
yes, seems redundant, but done.